docs/releases/1.10.txt

```diff
@@ -353,7 +353,11 @@ Backwards incompatible changes in 1.10
 Database backend API
 --------------------
 
-* ...
+* The GIS field ``AreaField`` uses an unspecified underlying numeric type,
```

Comment:
I don't know if this is important to mention ... will any database backends need adaption for this change? (I wonder if there are even any third-party GIS backends?)